### PR TITLE
Wait for signal from the sandbox

### PR DESCRIPTION
**Pull request by lbogdan**
Should be deployed after merging https://github.com/CompuIves/codesandbox-client/pull/417.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -99,6 +76,7 @@ export async function handler(event: any, context: Context, cb: Callback) {
     const page = await browser.newPage();
     await page.setViewport({ width: 1200, height: 630 });
     await loadPage(page, `https://${sandboxId}.codesandbox.io/`);
+    await page.waitFor(1000);
```
Comment (lbogdan) [inline src/index.ts]: Wait a bit for fonts, images etc. to load and the sandbox to run.

Comment (CompuIves) [reply src/index.ts]: I think we should make this 2s, just to be sure.

Comment (lbogdan) [reply src/index.ts]: Done.

Comment (CompuIves) [reply src/index.ts]: Whoop!

**Pull request merged**